packages/fx-core/src/plugins/solution/fx-solution/v2/executeUserTask.ts

```diff
@@ -398,11 +482,37 @@ export async function addCapability(
       pluginNamesToScaffold.add(ResourcePluginsV2.FrontendPlugin);
       if (!alreadyHasBot && !alreadyHasME) {
         pluginNamesToArm.add(ResourcePluginsV2.BotPlugin);
+
+        if (isAadManifestEnabled() && alreadyHasSso) {
+          const createAuthFilesRes = await createAuthFiles(inputsNew, false, true, true);
+          if (createAuthFilesRes.isErr()) {
```

Comment:
Seems return error logic are all same, can we extract to a function to make it simple

Reply:
Sure, will do